bbin

```diff
@@ -17,7 +17,9 @@
            org.babashka/http-client {:mvn/version "0.1.8"},
            version-clj/version-clj {:mvn/version "2.0.2"},
            babashka/fs {:mvn/version "0.3.17"},
-           org.babashka/json {:mvn/version "0.1.1"}}})
+           ring/ring-core {:mvn/version "1.13.0"},
+           org.babashka/json {:mvn/version "0.1.1"},
+           http-kit/http-kit {:mvn/version "2.8.0"}}})
```

Comment:
I didn't expect these dependencies to make it into `./bbin` - I propose to move these deps to the `:dev` alias to avoid adding depdendencies to `bbin` users.